RequestManagementSystem/ConfigTemplate.cfg

```diff
@@ -20,19 +24,23 @@ Services
 }
 Agents
 {
+  ##BEGIN RequestExecutingAgent
   RequestExecutingAgent
   {
     PollingTime = 60
-    RequestsPerCycle = 50
-    MinProcess = 1
-    MaxProcess = 8
-    ProcessPoolQueueSize = 25
+    # number of Requests to execute per cycle
+    RequestsPerCycle = 100
```

Comment:
Why did you double?

Reply:
That is what is in the code
https://github.com/DIRACGrid/DIRAC/blob/e80e60df88663927cb6b258920a86c1339a7b1f8/RequestManagementSystem/Agent/RequestExecutingAgent.py#L75-L76